onnxruntime/core/providers/coreml/builders/impl/split_op_builder.cc

```diff
@@ -139,8 +139,8 @@ bool SplitOpBuilder::IsOpSupportedImpl(const Node& node, const OpBuilderInputPar
     }
     const auto& splits_tensor = *initializers.at(input_defs[1]->Name());
     Initializer unpacked_tensor(splits_tensor);
-    auto splits_span = unpacked_tensor.DataAsSpan<uint64_t>();
-    int sum_of_splits = std::accumulate(splits_span.begin(), splits_span.end(), 0);
+    auto splits_span = unpacked_tensor.DataAsSpan<int64_t>();
+    int64_t sum_of_splits = std::accumulate(splits_span.begin(), splits_span.end(), uint64_t(0));
```

Comment:
why is the third argument `uint64_t(0)`?
```suggestion
    int64_t sum_of_splits = std::accumulate(splits_span.begin(), splits_span.end(), int64_t{0});
```

Reply:
good catch. missed reverting an update. my initial change was to make everything uint64_t but then I realized that the splits were actually `int64_t`.